src/components/forms/transaction/SplitField.tsx

```diff
@@ -40,6 +50,20 @@ export default function SplitField({
   if (splitCurrency && fromAccountCurrency !== splitCurrency) {
     showCurrencyExchange = true;
   }
+  const watchAccount = watch(`splits.${index}.toAccount`);
+
+  React.useEffect(() => {
```

Comment:
Basically, fetch and update exchange rate if any of these change:

- split account
- date
- split currency or fromAccount currency